rllib/algorithms/ppo/torch/ppo_torch_rl_module.py

```diff
@@ -222,24 +177,25 @@ def output_specs_inference(self) -> SpecDict:
 
     @override(RLModule)
     def _forward_inference(self, batch: NestedDict) -> Mapping[str, Any]:
-        shared_enc_out = self.shared_encoder(batch)
-        pi_enc_out = self.pi_encoder(shared_enc_out)
+        output = {}
 
-        action_logits = self.pi(pi_enc_out[ENCODER_OUT])
+        encoder_outs = self.encoder(batch)
+        output[STATE_OUT] = encoder_outs[STATE_OUT]
```

Comment:
I don't get why it gets complicated if we want to defer the support of RNNs to later. All you have to do is to massage the batch when passing it as input and also ignoring STATE_OUT keys when they you want to return the output. We can change this pattern to something more general once the RNN stuff are figured out.